.github/workflows/release.yaml

```diff
@@ -11,8 +11,8 @@ jobs:
     needs: image
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
         with:
           go-version: ${{ env.go-version }}
```

Comment:
Why don't you use `go-version-file: go.mod` here?

or

Why does this workflow still use go 1.16 ?